### PR TITLE
Creating a method for CreateParty 

### PR DESCRIPTION
**Pull request by brownrw8**
Per documented createParty endpoint: https://api-sandboxdash.norcapsecurities.com/admin_v3/documentation?mid=MTMy

Currently only contains required fields - this can be expanded upon.

Includes minor version bump for minor (non-breaking) dependencies via `npm audit fix`

### CODE REVIEW

File: src/createParty/enums.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const enum Domicile {
+  USCITIZEN = "U.S. Citizen",
```
Comment (shadowlion) [inline src/createParty/enums.ts:2]: Change to `US_CITIZEN`

File: src/createParty/enums.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  USRESIDENT = "U.S. Resident",
```
Comment (shadowlion) [inline src/createParty/enums.ts:3]: Change to `US_RESIDENT`

File: src/createParty/enums.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  NONRESIDENT = "non-resident",
```
Comment (shadowlion) [inline src/createParty/enums.ts:4]: Change to `NON_RESIDENT`

File: src/createParty/types.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+export interface CreatePartyResponse {
+  statusCode: string;
+  statusDesc: string;
+  partyDetails: PartyDetail[];
+}
+
+interface PartyDetail {
+  partyId: integer;
```
Comment (shadowlion) [inline src/createParty/types.ts:28]: Change type to `string` (party ID is normally something like "P12345")

**Review by shadowlion — CHANGES_REQUESTED** [review]
Please see comments for changes. Also, can you add a new line to all files as well? I know I didn't add it to `.prettierrc`, but it's gonna change anyways when I open it back on my setup.

**Comment by brownrw8** [comment]
How's it looking now?

**Review by shadowlion — APPROVED** [review]
lgtm - thanks!

**Pull request merged**